Cargo.toml

```diff
@@ -30,6 +30,9 @@ fastrace-opentelemetry = { path = "fastrace-opentelemetry" }
 # crates.io dependencies
 futures = { version = "0.3" }
 log = { version = "0.4" }
+opentelemetry = { version = "0.27", features = ["trace"] }
+opentelemetry_sdk = { version = "0.27", features = ["trace"] }
+opentelemetry-otlp = { version = "0.27", features = ["trace"] }
```

Comment:
it maybe better to declare otel dep in fastrace-opentelemetry?

Reply:
OK. I found it declare in both fastrace-otel and dev-deps of fastrace.  So lift it up to reduce dup and manual sync each time.

But push down makes sense in some points also. Let me revert it.